diode-server/sqlc.yaml

```diff
@@ -0,0 +1,12 @@
+version: "2"
```

Comment:
sqlc runs transparently at build time? if not, should we add this to Makefile?

Reply:
Added following recipes:
```
make install-dev-tools
make gen-mocks
make gen-dbstore
```

Reply:
great! thanks for that